src/dtx/dtx_common.c

```diff
@@ -284,6 +284,7 @@ dtx_handle_init(struct dtx_id *dti, daos_handle_t coh, daos_epoch_t epoch,
 
 	dth->dth_op_seq = 0;
 	dth->dth_oid_cnt = 0;
+	dth->dth_oid_cnt = 0;
```

Comment:
dup code?

Reply:
It should be "dth_rsrvd_cnt". Will fix it.